### ISSUE TITLE
Error creating migrations and Initial Create

### ISSUE THREAD

**Issue by rajat-vig** [aspnet/EntityFramework.Docs#1358]
Unable to create an object of type 'BloggingContext'. For the different patterns supported at design time, see https://go.microsoft.com/fwlink/?linkid=851728

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c78ab24c-ddc8-521d-f032-68b9dfaab803
* Version Independent ID: 23d99a0f-1eb8-3f0b-5a09-f2520aa3b496
* Content: [Getting Started on ASP.NET Core - New database - EF Core](https://docs.microsoft.com/en-us/ef/core/get-started/aspnetcore/new-db?tabs=netcore-cli#feedback)
* Content Source: [entity-framework/core/get-started/aspnetcore/new-db.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/get-started/aspnetcore/new-db.md)
* Product: **entity-framework**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by TheIncorrigible1** [comment]
Unable to replicate following the examples from the page.  Be sure that your project can build, otherwise running `database update` will fail.

Your link points to https://docs.microsoft.com/en-us/ef/core/miscellaneous/cli/dbcontext-creation

**Comment by ll01** [comment]
The documentation is out of date you need to change DbContext to the IDesignTimeDbContextFactory

**Comment by corix010** [comment]
I am also having issue with Add-Migration command in Package Manager Console. I'm getting the error "The specified deps.json [C:\App Dev Sample\DotNet\Sample5\SampleCoreMVC\SampleCoreMVC.deps.json] does not exist". The solution builds successfully. I tried several times but still getting the same error. I searched for solution to the error but cannot find solution that works.

I'm using Visual Studio Professional 2019 Version 16.1.6 on Windows 10 machine.
